includes/Server.hpp

```diff
@@ -21,43 +21,60 @@
 #include <sys/types.h>
 #include <unistd.h>
 
+#include <algorithm>
 #include <csignal>
 #include <cstdlib>
 #include <cstring>
 #include <iostream>
+#include <map>
 #include <sstream>
 #include <vector>
-#include <map>
-#include <algorithm>
 
-#include "../includes/Client.hpp"
 #include "../includes/Channel.hpp"
+#include "../includes/Client.hpp"
+
+typedef std::map<int, Client> clientsMap;
+typedef std::map<std::string, Channel> channelsMap;
+typedef Client &ClientRef; // Type de référence pour les clients
```

Comment:
It would be better to keep the original type (rather than using a `typedef`) in this case, as it improves code readability for everyone.